### PR TITLE
Make some resource PDFs a11y friendly

### LINKED ISSUE #399: Replace PDFs with accessibility-refreshed versions
There are two PDFs that have been refreshed for accessibility (mostly re-formatting logos). Links to the finalized docs below: 

Fact Sheet: Prepaid Card Recommendations - https://drive.google.com/file/d/1pHrA5jKdoPwZdJAFRKrfsDphu7GBDXOD/view?usp=drive_link

Overview: How Cal-ITP incorporated public feedback into CA Transit Data Guidelines v3.0 - https://drive.google.com/file/d/1T6JD67sW7Hsv5iwwQCwj80B_UPiojE3i/view?usp=drive_link

Note: No need to update the dates, these are not materially different in content they are just slightly re-formatted.

### PR DESCRIPTION
**Pull request by allejo**
Closes #399

### CODE REVIEW

File: src/_resources/slide-deck-ca-transit-data-guidelines-draft-v3.0.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 date: "2022-10-01T17:00:00-07:00"
 title: "Overview: How Cal-ITP incorporated public feedback into CA Transit Data Guidelines v3.0"
-asset: "Cal-ITP.Transit.Draft.Guidelines.Draft.3.Feedback.pdf"
+asset: "Cal-ITP.Transit.Draft.Guidelines.Feedback.pdf"
```
Comment (allejo) [inline src/_resources/slide-deck-ca-transit-data-guidelines-draft-v3.0.md:4]: @mrose914 is this change okay? The updated file you gave me had the updated name without the extra `Draft.3` so I kept that name. Just to let you know, if this PDF has been linked to by another site or page, this name change could break those links.

**Comment by thekaveman** [comment]
Sorry @allejo, I didn't realize @machikoyasuda wasn't part of the `web-review` team and this PR only asked for its review.

@machikoyasuda I added you to that team now. This team was originally set up so @mrose914 could approve simple content changes for go-live without an engineer.

File: src/_resources/slide-deck-ca-transit-data-guidelines-draft-v3.0.md (hunk 1 above)
Comment (machikoyasuda) [reply src/_resources/slide-deck-ca-transit-data-guidelines-draft-v3.0.md:4]: I checked on Google Search Console and other backlinks checkers to see if there are any websites that link to https://www.calitp.org/assets/Cal-ITP.Transit.Draft.Guidelines.Draft.3.Feedback.pdf. Then, I checked in the Cal-ITP Slack to see if this URL is linked or referenced there -- no references. So I feel comfortable changing this link. 

In an ideal future scenarios, we should reject URL filenames with Draft in them.

**Review by machikoyasuda — APPROVED** [review]

File: src/_resources/slide-deck-ca-transit-data-guidelines-draft-v3.0.md (hunk 1 above)
Comment (thekaveman) [reply src/_resources/slide-deck-ca-transit-data-guidelines-draft-v3.0.md:4]: Wait @machikoyasuda @allejo I think this was incorrect, or at least wasn't finished all the way. 

This links to a deck on Cal-ITP's work on the Draft v3 Transit Data Guidelines (e.g. the working version of v3 when it was in-progress but v2 was still current). This deck is highlighting feedback Cal-ITP took during the development of this v3 update.

In any case, `Draft` is still in the file name, but now we lost the `3` version number.

Comment (machikoyasuda) [reply src/_resources/slide-deck-ca-transit-data-guidelines-draft-v3.0.md:4]: Ooooh... oops. It looks like we are waiting on GDrive access to this PDF still?

Comment (machikoyasuda) [reply src/_resources/slide-deck-ca-transit-data-guidelines-draft-v3.0.md:4]: Wait. No. I see we do have the PDF here now: https://github.com/cal-itp/calitp.org/blob/6c97915c3d49326746d0bf04e2d77a60c967e0ca/src/assets/Cal-ITP.Transit.Draft.Guidelines.Feedback.pdf

I see the PDF itself has the title: CA Transit Data Guidlines Draft v3.0 Feedback, so it seems to me like that would be the best URL slug?

Comment (machikoyasuda) [reply src/_resources/slide-deck-ca-transit-data-guidelines-draft-v3.0.md:4]: Will make a new issue and PR to fix

**Comment by machikoyasuda** [comment]
Update: All fixed in this PR https://github.com/cal-itp/calitp.org/pull/405 - Thanks @mrose914 @allejo @thekaveman

**Pull request merged**